Gemfile.lock

```diff
@@ -221,6 +221,8 @@ GEM
       jekyll-feed (~> 0.9)
       jekyll-seo-tag (~> 2.1)
     minitest (5.16.1)
+    nokogiri (1.13.9-arm64-darwin)
```

Comment:
Could you please revert your changes on this file? Thanks

Reply:
The `gemfile.lock` was generated by the `gemfile` when I ran `bundle`, I didn't edit anything in the original `gemfile`. I also ran `gem install jekyll bundler` to install jekyll, maybe those came from there and are jekyll dependencies?

I think the `arm64` is only support for arm architecture.

@bebatut what would be the best way to go about this?

Reply:
could you revert your changes on this file and make sure you do not add this file to the commits?

Reply:
@bebatut it adds those every time I run the jekyll server

so if I needed to see something I would need to start the server, stop it, revert the changes, make my changes, add/commit, start again / stop / revert etc...

are those jekyll dependencies a problem?

I could work from gitlab, but that has a time limit for the free tier.. so I don't think that is much of a solution 🙁